aws_doc_sdk_examples_tools/config/sdks.yaml

```diff
@@ -295,6 +295,16 @@ Python:
       api_ref:
         uid: "boto3"
         name: "&guide-python3-api;"
+    301:
+      long: "&Python301long;"
+      short: "&Python301;"
+      expanded:
+        long: "AWS SDK for Python Async"
+        short: "SDK for Python Async"
+      guide: "https://github.com/awslabs/aws-sdk-python"
```

Comment:
The guide is currently at `https://github.com/awslabs/aws-sdk-python/blob/develop/dev-guide/index.md`. It will move to the docs site when we add the next service to the SDK sometime late in the summer or early in the autumn.